src/front/glsl_new/parser.rs

```diff
@@ -41,7 +41,24 @@ pomelo! {
     %type function_definition Function;
 
     root ::= version_pragma translation_unit;
-    version_pragma ::= Version IntConstant Identifier?;
+    version_pragma ::= Version IntConstant(V) Identifier?(P) {
+        match V.1 {
+            450 => (),
+            _ => return Err(ErrorKind::InvalidVersion(V.0, V.1))
+        }
+        extra.version = V.1 as u16;
+        match P {
```

Comment:
let's do `extra.profile = match P {..};`

Reply:
still needs to be done?

Reply:
Also done in latest commit

Reply:
which isn't pushed?

Reply:
weird, visible to me: https://github.com/gfx-rs/naga/pull/109/commits/914d3d78e22610463042383d253f5fe8f5ca6849#diff-34be000f1f7510bb72876c176236e89eR54

Reply:
can we move it out of the outer match?

Reply:
done